bundle/View/Redirect/Resolver.php

```diff
@@ -84,11 +84,7 @@ private function processUrl(RedirectConfiguration $redirectConfig): string
     {
         $url = $redirectConfig->getTarget();
 
-        $queryStringArray = [];
-        foreach ($redirectConfig->getTargetParameters() as $key => $value) {
-            $queryStringArray[] = \urlencode($key) . '=' . \urlencode($value);
-        }
-
-        return \count($queryStringArray) > 0 ? $url . '?' . \implode('&', $queryStringArray) : $url;
+        return \count($redirectConfig->getTargetParameters()) > 0 ?
+            $url . '?' . \http_build_query($redirectConfig->getTargetParameters()) : $url;
```

Comment:
Final nitpick, I promise :)

What if the URL already has query params?

Reply:
After internal discussion, it was decided to simply skip using `target_parameters` in case an absolute URL was provided. Absolute URL can contain query parameters and/or a fragment, which would only complicate adding custom query params taken from `target_parameters`.